include/net/fota_download.h

```diff
@@ -172,14 +171,15 @@ int fota_download(const char *host, const char *file, const int *sec_tag_list,
  *              download, both paths will be treated as upgradable bootloader slot 0
  *              and slot 1 binaries respectively, and only the binary corresponding to
  *              the currently inactive bootloader slot will be selected and downloaded.
-*               See <a href="https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/bootloaders/bootloader.html">
+ *              See <a href="https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/
+ *              config_and_build/bootloaders/bootloader.html">
```

Comment:
This will create a link to:
`https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/%20%20%20%20%20%20%20%20%20%20%20%20%20config_and_build/bootloaders/bootloader.html`

Which naturally doesn't exist. I don't think you can put a line break in the middle of a link.